### DISCUSSION TITLE
Endpoint para upload de arquivo (Token AntiForgery)

### DISCUSSION THREAD

**Discussion by FBODeveloper** [balta-io/forum#635] [Tira Dúvidas] [answered]
Estou criando um endpoint para upload de arquivos, criar até criei mas usando o Identity como incluir no token no Cookie o token antiforgery?
Alguém já fez um endpoint de upload e poderia ajudar?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @FBODeveloper, como vai?

Segue a implementação
https://learn.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-8.0

**Reply by FBODeveloper** [reply]
vou tentar.... to achando meio complexo...
valeu!!!
